pkg/config/fetcher/tracers/from_application_monitoring.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+// Package tracers is a collection of helpers to fetch tracer configurations from the disk.
+package tracers
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/DataDog/datadog-agent/comp/core/config"
+)
+
+// ApplicationMonitoringConfig fetches the `application_monitoring.yaml` configurations from the disk.
+func ApplicationMonitoringConfig(config config.Reader) (string, error) {
+	return readConfigFile(config, "application_monitoring.yaml")
```

Comment:
Who is using this file ? Seems like the Agent never load them. I'm trying to understand if the agent metadata payload is the best place for this.

Also how big are the files ?